examples/asset/utilization/AssetUtilization.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NationalInstruments.SystemLink.Clients.AssetManagement;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Asset.Utilization
+{
+    /// <summary>
+    /// Example for the SystemLink Asset Management client API that
+    /// tracks utilization for assets present in the system.
+    /// 
+    /// Note: As a prerequisite for this example, ensure SystemLink Client is installed.
+    /// </summary>
+    class AssetUtilization
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * The Asset Utilization Store enables asset utilization tracking. 
+             * It is created using a factory.
+             */
+            var utilizationStoreFactory = new AssetUtilizationStoreFactory();
+            var assetUtilizationStore = utilizationStoreFactory.CreateAssetUtilizationStore();
+
+            /*
+             * Tracking Asset Utilization can be configured to send heartbeats
+             * for ongoing utilization sessions, at predefined intervals. 
+             * By default, heartbeats are automatically sent every 5 minutes.
+             */
+            var utilizationConfiguration = StartUtilizationConfiguration.CreateDefault();
+
+            /*
+             * Start an utilization session for all assets in the system.
+             * For starting an utilization session, the following information is nedeed:
+             * - the utilization configuration,
+             * - [optional] asset names: a single asset name or a collection of asset names. if this parameter is ommited, all assets are tracked.
```

Comment:
A few things:

- Can we capitalize "The" in the first bullet and remove the comma?
Note: Can we capitalize all the first words in each listed item? If you rather keep them lower-cased, then we should remove all articles preceding the parameter names, such as "the". 
- "[Optional] asset name(s):"? 
- Can we capitalize "If"?
- "omitted"